src/components/VersionRestartModal/VersionRestartModal.tsx

```diff
@@ -208,4 +289,15 @@ const Row = styled.div`
   }
 `;
 
+export const TitleContainer = styled.div`
+  margin-top: 15px;
+  width: 96%;
+`;
+
+const BoldTextStyle = styled.div`
```

Comment:
[nit] Since this is an inline text element, `span` is more appropriate
```suggestion
const BoldTextStyle = styled.span`
```

Reply:
done